The function get_arch() return the target arch which blivet runs on. For yocto
x86 arches(qemux86 & genericx86), the target arch has been updated with 'i686'.
already. It will get false result when compare with host target. So tweak it
with 'i686' for yocto.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 blivet/arch.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blivet/arch.py b/blivet/arch.py
index 2cd978e..c66bbba 100644
--- a/blivet/arch.py
+++ b/blivet/arch.py
@@ -341,7 +341,7 @@ def get_arch():
 
     """
     if is_x86(bits=32):
-        return 'i386'
+        return 'i686'
     elif is_x86(bits=64):
         return 'x86_64'
     elif is_ppc(bits=32):
